schemas/blog-post.json

```diff
@@ -0,0 +1,15 @@
+{
+    "title": "Blog Post",
+    "slug": "blog_post",
```

Comment:
We decided on `SubjectType` enum to have dash as opposed to underscore earlier, we should keep both consistent, whatever that is. Personally, I think dash is better in all contexts where we might display it to the user or have it in the url?

Reply:
Ok, I've made it so that dashes are accepted (underscores are not), and I've limited the length to 16 characters. This would mean that the `post_type` prefix that we use must not exceed 4 characters, for example `twp_`.